RecoLocalCalo/Configuration/python/hcalGlobalReco_cff.py

```diff
@@ -24,10 +24,27 @@
 
 #--- for Run 3 on GPU
 from Configuration.ProcessModifiers.gpu_cff import gpu
+from Configuration.ProcessModifiers.alpaka_cff import alpaka
 
 from RecoLocalCalo.HcalRecProducers.hcalCPURecHitsProducer_cfi import hcalCPURecHitsProducer as _hbherecoFromCUDA
 (run3_HB & gpu).toModify(hbhereco,
     cuda = _hbherecoFromCUDA.clone(
         produceSoA = False
     )
 )
+
+from RecoLocalCalo.HcalRecProducers.hcalRecHitSoAToLegacy_cfi import  hcalRecHitSoAToLegacy 
+(alpaka & run3_HB).toModify(hbhereco,
+    cpu = hcalRecHitSoAToLegacy.clone(
+        recHitsM0LabelIn = ("hbheRecHitProducerPortable","")
+    ),
+    cuda = hcalRecHitSoAToLegacy.clone(
+        recHitsM0LabelIn = ("hbheRecHitProducerPortable","")
+    )
```

Comment:
Is this one needed ?
In principle we should not use both `gpu` and `alpaka` modifiers at the same time - and without the `gpu` modifier, the `cuda` branch should not be set ?

Reply:
Removed in 97d3bc4114273cc29e8f9ad94713d3adc10c7a5a